cjworkbench/management/commands/load_socialaccounts.py

```diff
@@ -0,0 +1,38 @@
+# Startup code
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.sites.models import Site
+from cjworkbench import settings
+
+class Command(BaseCommand):
+    help = 'Loads the social accounts (with secrets) from the config file'
```

Comment:
If I saw this message or found this file I don't think I would connect "social accounts" with login options. Can you add a brief comment to describe what we are doing here, and why it's a manage command (your 
 hard won knowledge)?